whos-online/whos-online.js

```diff
@@ -0,0 +1,56 @@
+friends = [
```

Comment:
I guess this variable is used for testing purposes, but be aware that it can confuse you about where and when put some functions inside or outside `whosOnline`, as mentioned below.

Reply:
Yup, that kinda confused me. Moved everything again. The only function staying out of `whosOnline` is the `getUsername` one.